Encounter1.sln

```diff
@@ -7,16 +7,104 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Encounter1", "Encounter1\En
 EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Ad-Hoc|Any CPU = Ad-Hoc|Any CPU
```

Comment:
Why are all these lines added to Encounter1.sln file?

Reply:
I was trying to add iOS to our app but it didn't work out for me. 
I will remove these lines quickly